### PR TITLE
Vet clinic/create animals table

### PR DESCRIPTION
**Pull request by trace-kadenyi**
- Create database named `vet_clinic`
- Create table named `animals`
- Insert pertinent data into the table
- Make queries for certain information
- Below are two screenshorts of the query results.

SQL Queries - part 1
SQL Queries - part 2

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,17 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon';
+
+SELECT * FROM animals WHERE date_of_birth BETWEEN date '01-01-2016' AND date '01-01-2019';
```
Comment (sja-thedude) [inline queries.sql]: - [x] Kindly make sure that the end date is `'2019/31/12' `as per the requirements.
- [x] Also make sure to display the screenshot for this as the previous one was not right.

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @TracK92, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Professional Readme
✔️ Correct gitflow

## Required 
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (trace-kadenyi) [reply queries.sql]: Hi [sja-thedude](https://github.com/sja-thedude),
I'm a bit confused about your requirements as I did not see the requirement for the date format. Kindly direct me to it. Thanks.

**Comment by trace-kadenyi** [comment]
Hello, 

I have made the changes and uploaded new screenshots. Thanks.
SQL Queries - part 1

SQL Queries - part 2

**Review by Reem-lab — APPROVED** [review]
# Hi @TracK92,👋🏻👋🏻

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉🔥🔥💯💯



## Highlights ✨✨
📌 Descriptive pull request✅
📌 Great working on the functionality✅
📌 Good commit messages ✅
📌 Professional README ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**